CONTRIBUTING.md

```diff
@@ -68,4 +68,13 @@ you can use one of some generic reasons like "Improve documentation.",
 
 The project now enables [DCO check](https://github.com/probot/dco#how-it-works) and the commit message must contain a `Signed-off-by` line for [Developer Certificate of Origin](https://developercertificate.org/)
 
-You can use option `-s` for `git commit` to automatically add a `Signed-off-by` to the commit message. 
\ No newline at end of file
+You can use option `-s` for `git commit` to automatically add a `Signed-off-by` to the commit message. 
+
+### Handling CI
+
+We use an ad-hoc jenkins CI deployment to test each PR. If you are submitting a
+PR for the first time, you will need someone of us to comment`/ok-to-test` to
+trigger the CI. After this, new changes on the branch will be able to trigger
+the test automatically. You may also manually run the test by commenting
+`/test`.
```

Comment:
@zhouqiang-cl Is `/rebuild` and `/test` the same?

Reply:
Tried `/rebuild`, seems to be the same.